packaging/examples/metrics/grafana-dashboards/strimzi-kafka-connect.json

```diff
@@ -263,9 +263,9 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(kafka_consumer_incoming_byte_total{strimzi_io_kind=~\"KafkaConnect.*\",strimzi_io_cluster=\"$strimzi_connect_cluster_name\"}[1m])) by (kubernetes_pod_name)",
+          "expr": "sum(rate(kafka_consumer_incoming_byte_total{strimzi_io_kind=~\"KafkaConnect.*\",strimzi_io_cluster=\"$strimzi_connect_cluster_name\"}[1m])) by (pod)",
           "interval": "",
-          "legendFormat": "{{kubernetes_pod_name}}",
+          "legendFormat": "{{pod}}",
```

Comment:
From my POV yes, at least I didn't see any issues when I applied these changes to my grafana dashboards. I think that if we will keep `kubernetes_pod_name`, the colours will be different in this panel, or at least it was during my testing.

Reply:
> I think that if we will keep kubernetes_pod_name, the colours will be different in this panel, or at least it was during my testing.

The question would be ... why? What's the difference between these two attributes? Are you trying on OpenShift or Kubernetes? I think `kubernetes_pod_name` is coming from Prometheus but if on OpenShift maybe you are relaying on its own Prometheus instance that could add `pod` attribute as well? I am just guessing trying to understand the difference between the two.

Reply:
I am trying it on OpenShift. Monitoring stack available with OpenShift installation does not inject `kubernetes_pod_name` for all available metrics we show in the dashboards. You can check for example `container_cpu_usage_seconds_total` metric which is shown in KC or MM2 dashboards. This query does not return `kubernetes_pod_name`, guess this was also the reason for changes in https://github.com/strimzi/strimzi-kafka-operator/pull/4149/ from `kubernetes_pod_name` to `pod`. I will try to double-check it on Kubernetes cluster, but I assume it will work as it works on OpenShift.